test/arrays.js

```diff
@@ -0,0 +1,117 @@
+var assert = require('assert');
+var diff = require('../index.js');
+
+describe('Array Diff', function() {
+  it('both children edit same key to same array values', function() {
```

Comment:
Sorry, I meant that the children aren't editing anything in the code of this test (if they were, I'd be expecting them to be different from the parent)